BeeTracker/app/src/main/res/layout/review_fragment.xml

```diff
@@ -1,19 +1,114 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:orientation="vertical" android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <TextView
-        android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#F5F5F5">
+<RelativeLayout
```

Comment:
Please fix the spacing here, can you also copy the layout used in identify_fragment and camera_fragment from my branch? It looks a lot closer to the spec